.github/workflows/release.yaml

```diff
@@ -3,7 +3,7 @@ name: Release Charts
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
i did not see this branch in the current repo, am i missing something?

Reply:
@carlisia We can make the branch in this repo now and edit this PR.